Changelog.md

```diff
@@ -1,7 +1,7 @@
 ### 0.8.29 (unreleased)
 
 Language Features:
- * Introduce syntax for specifying contract storage layout base.
+ * Introduce syntax and code generation for custom contract storage layout base.
```

Comment:
It's a complete feature now, not need to split hairs about syntax or code generation. The entry should just say what it does.

```suggestion
 * Allow relocating contract storage to an arbitrary location.
```

Reply:
Done.